pkgs/by-name/om/omnicli/package.nix

```
@@ -0,0 +1,52 @@
+{
+  fetchFromGitHub,
+  lib,
+  openssl,
+  perl,
+  pkg-config,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "omnicli";
+  version = "2025.4.0";
+
+  src = fetchFromGitHub {
+    owner = "xaf";
+    repo = "omni";
+    tag = "v${finalAttrs.version}";
+    sha256 = "sha256-NRazjD20mmpe8dOoJNu8e/H8IRRkf/MLGKIxbw26Vbk=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-dn5epW07HO6c4sSNhhg4oZjPPYWKkxG+a7i1CrQNcok=";
+
+  nativeBuildInputs = [
+    pkg-config
+    perl
+  ];
+
+  buildInputs = [ openssl ];
+
+  patchPhase = ''
+    substituteInPlace Cargo.toml \
+      --replace 'version = "0.0.0-git"' 'version = "${finalAttrs.version}"'
+    substituteInPlace Cargo.lock \
+      --replace 'version = "0.0.0-git"' 'version = "${finalAttrs.version}"'
+  '';
+
+  checkPhase = ''
+    cargo test -- \
+      --skip internal::cache::up_environments::tests::up_environment::test_new_and_init \
+      --skip internal::config::up::cargo_install::tests::install \
+      --skip internal::config::up::github_release::tests::up
+  '';
+
+  meta = {
+    homepage = "https://omnicli.dev/";
+    description = "The omnipotent dev tool";
```

```suggestion
    description = "Omnipotent dev tool";
```
See [README.md#meta-attributes](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes)

> Not start with the definite or an indefinite article.